hazelcast/src/test/java/com/hazelcast/config/XMLConfigBuilderTest.java

```diff
@@ -700,6 +700,33 @@ public void readReliableTopic() {
         assertNull(listenerConfig.getImplementation());
     }
 
+    @Override
+    @Test(expected = InvalidConfigurationException.class)
+    public void testCaseInsensitivityOfSettings() {
+        String xml = HAZELCAST_START_TAG
+                + "<map name=\"testCaseInsensitivity\">"
+                + "    <in-memory-format>BINARY</in-memory-format>"
+                + "    <backup-count>1</backup-count>"
+                + "    <async-backup-count>0</async-backup-count>"
+                + "    <time-to-live-seconds>0</time-to-live-seconds>"
+                + "    <max-idle-seconds>0</max-idle-seconds>    "
+                + "    <eviction eviction-policy=\"NONE\" max-size-policy=\"per_partition\" size=\"0\"/>"
+                + "    <merge-policy batch-size=\"2342\">CustomMergePolicy</merge-policy>"
+                + "</map>"
+                + HAZELCAST_END_TAG;
+
+        Config config = buildConfig(xml);
```

Comment:
It should be inline with ICache now, but yes this discrepancy exists between xml and yaml. Also, now, since we use xsd-enumaration, case sensitive-ness should not cause much hassle, before we were using free text for map max size policy.